packages/aws-otel-collector/aws-otel-collector-default-config.yaml

```diff
@@ -0,0 +1,37 @@
+extensions:
+  health_check:
```

Comment:
Not entirely sure it makes sense to ship a config at all here, but if we do then it should be a bare minimum - just the most basic one possible with all optional functionality turned off.

The awsxray, awsemf, http stuff I would cut.

Downstream variants can pretty easily specify a different config file via systemd unit drop-in that overrides the ExecStart command to point to their custom config or rendered template.

Reply:
I believe the collector requires some sort of config which specifies receivers, processors, and exporters; otherwise it will fail. I tried creating a patch to make the config optional but it causes the collector to fail with invalid configuration errors with [ResolverSettings](https://github.com/aws-observability/aws-otel-collector/blob/311c612da46c22aa0e32d3787dd422bbaba8e83d/pkg/config/config_factory.go#L57), which is [an import from the general OTEL collector pkg. ](https://github.com/open-telemetry/opentelemetry-collector/blob/confmap/v0.106.1/confmap/resolver.go#L33)

I've created a minimalist config based on possible core (non-aws specific) components which the collector still functions with here: https://gist.github.com/koooosh/8cddd42a91657552af4709b5d1157bef

Let me know if you notice anything off-putting in that.